### ISSUE TITLE
-X flag requires argument of the form importpath.name=value

### ISSUE THREAD

**Issue by sint** [facette/facette#307]
I've just tried to compile facette and i get this error:

```
facette: Building facette...
# command-line-arguments
/usr/lib/go/pkg/tool/linux_amd64/link: -X flag requires argument of the form importpath.name=value
result: fail
Makefile:116: recipe for target 'build/facette-linux-amd64/bin/facette' failed
make: *** [build/facette-linux-amd64/bin/facette] Error 1
```

Can anyone please help me? I am using latest released version with go 1.7.1

**Comment by vbatoufflet** [comment]
Hi @sint,

The issue was addressed by the 7ae8cc2e337eb047c2939cd86949a0003bdd0d3f change which is only available on `master`, not in the releases.

**Comment by sint** [comment]
Thanks allot. It works! Are there any plans add this fix into a new release?

**Comment by vbatoufflet** [comment]
An alpha release of the 0.4.0 version if planned to land soon. It contains an almost fully rewritten backend and UI.

**Comment by vbatoufflet** [comment]
Addressed in [0.4-alpha](https://github.com/facette/facette/tree/0.4-alpha).
